specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1795 @@
+        "operationId": "GroupQuotaSubscriptionQuotaAllocation_CreateOrUpdate",
+        "x-ms-examples": {
+          "SubscriptionQuota-Allocation_Put_Request_ForCompute": {
+            "$ref": "./examples/SubscriptionQuota-Allocation/SubscriptionQuota-Allocation_Put_Request-Compute.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/ResourceInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "allocateQuotaRequest",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SubscriptionQuotaAllocationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. , Create or Update",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionQuotaLimit"
+            }
+          },
+          "201": {
+            "description": "Request Created. Create or Update",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer",
+                "format": "int64"
+              },
+              "Location": {
+                "description": "URL for determining when an operation has completed. Send a GET request to the URL in Location header.\nThe URI should return a 202 until the operation reaches a terminal state and 200 once it reaches a terminal state.\n\nFor more info: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "URL for checking the ongoing status of the operation.\nTo get the status of the asynchronous operation, send a GET request to the URL in Azure-AsyncOperation header value.\n\nFor more info: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/OperationsStatus"
```

Comment:
>OperationsStatus [](http://example.com/codeflow?start=37&length=16)

This schema should represent the model of the resource not an operation status. Please check all Puts

Reply:
PutRequestResponseSchemeArm 

A PUT operation request body schema should be the same as its 200 response schema, to allow reusing the same entity between GET and PUT. If the schema of the PUT request body is a superset of the GET response body, make sure you have a PATCH operation to make the resource updatable. Operation: 'GroupQuotaSubscriptionQuotaAllocation_CreateOrUpdate' Request Model: 'parameters[6].schema' Response Model: 'responses[200].schema'
Location: Microsoft.Quota/preview/2023-06-01-preview/groupquota.json#L844

[https://github.com/Azure/azure-rest-api-specs/pull/23990/checks?check_run_id=15217443253](https://github.com/Azure/azure-rest-api-specs/pull/23990/checks?check_run_id=15217443253)

Reply:
@rkmanda we've resolved the PutRequestResponseSchemeArm error. is there anything else we should address?

Reply:
This is still not fixed.

Reply:
@visingla-ms 
<img width="893" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/67396382/a5f2e39c-6d8c-4a44-822c-845b299474ef">
these are the LintDiff errors that I'm seeing pop up at the moment. as far as i can tell the `PutRequestResponseSchemeArm` issue is fixed.

is there some other view i should be looking at perhaps?

Reply:
With 201 it should return the same schema with the appropriate ProvisioningState, and user will use the header to check the status. Here it should return model of the resource, see [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creatingupdating-using-put)

Reply:
updated put 201 returns. let me know if further action is required